src/mixins/filing-template-mixin.ts

```diff
@@ -691,6 +746,63 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     this.storeStaffPayment(filing)
   }
 
+  /**
+   * Parses a draft special resolution filing into the store.
+   * @param filing the special resolution filing
+   * @param entitySnapshot the latest entity snapshot
+   */
+  parseSpecialResolutionFiling (filing: SpecialResolutionFilingIF, entitySnapshot: EntitySnapshotIF): void {
+    // store Entity Snapshot
+    this.setEntitySnapshot(entitySnapshot)
+
+    // store Entity Type
+    this.setEntityType(filing.alteration.business?.legalType || entitySnapshot.businessInfo.legalType)
+
+    // store Business Information
+    this.setBusinessInformation({
+      ...filing.business,
+      ...filing.alteration.business
+    })
+
+    // store Name Request data
+    this.setNameRequest(
+      filing.alteration.nameRequest ||
+      {
+        legalType: entitySnapshot.businessInfo.legalType,
+        legalName: entitySnapshot.businessInfo.legalName,
+        nrNumber: entitySnapshot.businessInfo.nrNumber
+      }
+    )
+    //  add more components later
+
+    // store Office Addresses **from snapshot** (because we don't change office addresses in an special resolution)
+    this.setOfficeAddresses(entitySnapshot.addresses)
+
+    // store People And Roles **from snapshot** (because we don't change people and roles in an special resolution)
+    this.setPeopleAndRoles(entitySnapshot.orgPersons)
+
+    // store Business Contact
+    this.setBusinessContact(entitySnapshot.authInfo.contact)
+
+    // store Certify State
+    this.setCertifyState({
+      valid: false,
+      certifiedBy: filing.header.certifiedBy
+    })
+
+    // store Folio Number
+    // *** FUTURE: should we store correction.folioNumber instead?
+    this.setFolioNumber(entitySnapshot.authInfo.folioNumber || '')
+
+    // if Transactional Folio Number was saved then store it
+    if (filing.header.isTransactionalFolioNumber) {
+      this.setTransactionalFolioNumber(filing.header.folioNumber)
+    }
+
+    // store Document Optional Email
+    this.setDocumentOptionalEmail(filing.header.documentOptionalEmail)
+  }
```

Comment:
You will need to store the Staff Payment data from a draft filing (if staff can complete this filing).

Reply:
As far as design, there is no staff version now. I hope it will come soon, So i removed it now and when we have design we can add it back

Reply:
Sounds good.

And you can delete `buildStaffPayment` in the build method so it doesn't set values incorrectly.